plugins/obs-transitions/transition-stinger.c

```diff
@@ -87,6 +89,8 @@ static void stinger_update(void *data, obs_data_t *settings)
 	dstr_free(&name);
 	obs_data_release(media_settings);
 
+	if (s->media_source && s->transitioning)
+		obs_source_add_active_child(s->source, s->media_source);
```

Comment:
Prefer an empty line after an unenclosed `if`, or else enclose the if with curly braces.